src/target/riscv/riscv.h

```diff
@@ -314,9 +314,9 @@ struct riscv_info {
 	bool riscv_ebreaks;
 	bool riscv_ebreaku;
 
-	bool enable_equality_match_trigger;
-	bool enable_napot_trigger;
-	bool enable_ge_lt_trigger;
+	bool wp_enable_equality_match_trigger;
+	bool wp_enable_napot_trigger;
+	bool wp_enable_ge_lt_trigger;
```

Comment:
I'd recommend to use the word `allow` instead of `enable` (same as in the openocd.texi documentation) as it better describes the purpose of these bool flags.

```suggestion
	bool wp_allow_equality_match_trigger;
	bool wp_allow_napot_trigger;
	bool wp_allow_ge_lt_trigger;
```

Reply:
Addressed.